applications/FluidDynamicsApplication/python_scripts/check_and_prepare_model_process_fluid.py

```diff
@@ -63,6 +63,10 @@ def Execute(self):
         fluid_computational_model_part.AddConditions(list(list_of_ids))
 
         #verify the orientation of the skin (only implemented for tris and tets)
+        if self.main_model_part.NumberOfElements() == 0:
+            # Empty modelpart
+            return
+
         geometry = self.main_model_part.GetElement(1).GetGeometry()
```

Comment:
This will crash if you don’t have element with Id 1 in this partition
Please check the fluid solver how to get the first element (sth with __next__)

Reply:
True. Fixed.